README.adoc

```diff
@@ -268,26 +275,26 @@ and `system/src/main/liberty/config/server.xml` files:
 <feature>usr:opentracingZipkin-0.30</feature>
 ----
 
-Make sure your services are running, then simply point your browser to any of their endpoints and
+Make sure your services are running. Then simply point your browser to any of their endpoints and
 check your Zipkin server for traces.
 
 
 === Enabling explicit distributed tracing
 
-The `@Traced` annotation can be used to define explicit span creation for specific classes and methods.
-If the annotation is placed on a class, then the annotation is automatically applied to all methods
-within that class. If the annotation is placed on a method, then it overrides the class annotation
-if one exists.
+Use the `@Traced` annotation to define explicit span creation for specific classes and methods.
+If you place the annotation on a class, then its automatically applied to all methods within that class.
```

Comment:
I think we want "it's" with the apostrophe in "...it's automatically applied..."

Reply:
fixed